templates/apiary-bucket-policy.json

```diff
@@ -39,7 +39,11 @@
             "Sid": "Apiary customer account object permissions",
             "Effect": "Allow",
```

Comment:
I am unsure if adding this condition inside the text block is possible. The rest of the blocks add `if/else/for` at the beginning and at the end only.